pyccel/ast/builtin_methods/list_methods.py

```diff
@@ -169,3 +170,80 @@ def list_variable(self):
         Provide the name of the list as the return value.
         """
         return self._list_variable
+
+#=======================================================================================
+class ListInsert(PyccelInternalFunction):
+    """
+    Represents a call to the .insert() method.
+
+    Represents a call to the .insert() method of an object with a list type,
+    which inserts a given element at a given index in a list. This method returns `None`.
+    The insert method is called as follows:
```

Comment:
Same comment. Please stay within 80 characters per line, unless this makes the code unreadable.

Reply:
fixed in 12cd0b2a08f4f7d59e7c61e94769a7a9d9092039